test/integration/calculators/part_year_profit_tax_credits_calculator_test.rb

```diff
@@ -6,42 +6,43 @@ class PartYearProfitTaxCreditsCalculatorTest < ActiveSupport::TestCase
       setup do
         @calculator = Calculators::PartYearProfitTaxCreditsCalculator.new
 
-        @calculator.tax_credits_award_ends_on  = Date.parse('2015-08-01')
+        @calculator.tax_credits_award_ends_on  = Date.parse('2018-08-01')
```

Comment:
I'm not quite sure why the year has changed in this test? do we need to update the `context` labels as well?

Reply:
Ah shoot, you're right. This was supposed to be generic changes, but could update it for the most recent period as well

Reply:
Or I could change it back so it fits the old format and leave it to someone else to update the next time we change this?

Reply:
Could we leave this one as 2015 and create another test for 2018? Or will that just make it worse if we're copying and pasting a lot of tests?

Reply:
I don't think that's a good idea. This is just to improve the tests so there's fewer places to change the numbers when the tests change, and fewer values hard-coded.

Reply:
@thomasleese what do you think?

Reply:
@jonodrew I think we can close this one for now and Platform Health will have a think about how best to do it next quarter.